src/main/java/com/paloaltonetworks/osc/api/PANApplianceManagerApi.java

```diff
@@ -1,294 +1,286 @@
-     */
-    @Override
-    public IscJobNotificationApi createIscJobNotificationApi(ApplianceManagerConnectorElement mc,
-            VirtualSystemElement vs) throws Exception {
-
-        return null;
-    }
-
-}
+/* Copyright 2016 Palo Alto Networks Inc.
+ * All Rights Reserved.
+ *    Licensed under the Apache License, Version 2.0 (the "License"); you may
+ *    not use this file except in compliance with the License. You may obtain
+ *    a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ *    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ *    License for the specific language governing permissions and limitations
+ *    under the License.
+ */
+package com.paloaltonetworks.osc.api;
+
+import static org.osc.sdk.manager.Constants.*;
+
+import java.security.SecureRandom;
+
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.TrustManager;
+import javax.net.ssl.X509TrustManager;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+
+import org.osc.sdk.manager.api.ApplianceManagerApi;
+import org.osc.sdk.manager.api.IscJobNotificationApi;
+import org.osc.sdk.manager.api.ManagerCallbackNotificationApi;
+import org.osc.sdk.manager.api.ManagerDeviceApi;
+import org.osc.sdk.manager.api.ManagerDeviceMemberApi;
+import org.osc.sdk.manager.api.ManagerDomainApi;
+import org.osc.sdk.manager.api.ManagerPolicyApi;
+import org.osc.sdk.manager.api.ManagerSecurityGroupApi;
+import org.osc.sdk.manager.api.ManagerSecurityGroupInterfaceApi;
+import org.osc.sdk.manager.api.ManagerWebSocketNotificationApi;
+import org.osc.sdk.manager.element.ApplianceManagerConnectorElement;
+import org.osc.sdk.manager.element.VirtualSystemElement;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.metatype.annotations.AttributeDefinition;
+import org.osgi.service.metatype.annotations.ObjectClassDefinition;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.paloaltonetworks.panorama.api.methods.JAXBProvider;
+import com.paloaltonetworks.panorama.api.methods.ShowOperations;
+
+@Component(configurationPid = "com.paloaltonetworks.panorama.ApplianceManager",
+        property = { PLUGIN_NAME + "=Panorama", VENDOR_NAME + "=Palo Alto Networks", SERVICE_NAME + "=Panorama",
+                EXTERNAL_SERVICE_NAME + "=Pan-nsx", AUTHENTICATION_TYPE + "=BASIC_AUTH", NOTIFICATION_TYPE + "=NONE",
+                SYNC_SECURITY_GROUP + ":Boolean=true", PROVIDE_DEVICE_STATUS + ":Boolean=false",
+                SYNC_POLICY_MAPPING + ":Boolean=false", SUPPORT_MULTIPLE_POLICIES + ":Boolean=false" })
+public class PANApplianceManagerApi implements ApplianceManagerApi {

```

Comment:
Should the classes be renamed as well?

Reply:
They could but since the user does not see the class information...dont see a pressing reason. I believe dmity is doing some refactoring and renaming as part of his changes.